e2e/controls.py

```diff
@@ -2336,3 +2336,188 @@ def expect_n_row(
             n,
             timeout=timeout,
         )
+
+
+class _CardBodyP(_InputBaseP, Protocol):
+    loc: Locator
+
+
+class _CardBodyM:
+    def expect_body_to_contain_text(
```

Comment:
If we do not have other methods to qualify against, we should use smaller API names when possible

```suggestion
    def expect_body(
```